Extending/version7-assets.md

```diff
@@ -38,7 +38,7 @@ A handler for this method could look like:
 
 ## URLs
 
-A good rule of thumb about service URLs is to not hard code them if possible. One of the reasons why we don't hard code URLs is in case that routing has to change for some reason (i.e. [the breaking change for 7.0.2](https://umbraco.com/blog/heads-up-breaking-change-coming-in-702-and-62/)).
+A good rule of thumb about service URLs is to not hard code them if possible. One of the reasons why we don't hard code URLs is that in case the routing has to change for some reason (i.e. [the breaking change for 7.0.2](https://umbraco.com/blog/heads-up-breaking-change-coming-in-702-and-62/)).
```

Comment:
```suggestion
A good rule of thumb about service URLs is to not hard code them if possible. One of the reasons why we don't hard code URLs is in case the routing has to change for some reason (i.e. [the breaking change for 7.0.2](https://umbraco.com/blog/heads-up-breaking-change-coming-in-702-and-62/)).
```

Reply:
I don't see any changes to the files, are you sure this is resolved? 🙂